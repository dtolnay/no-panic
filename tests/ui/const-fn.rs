use no_panic::no_panic;

#[no_panic]
const fn f() {}

fn main() {}
