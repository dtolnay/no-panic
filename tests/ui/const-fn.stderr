error[E0015]: cannot call non-const closure in constant functions
 --> tests/ui/const-fn.rs:3:1
  |
3 | #[no_panic]
  | ^^^^^^^^^^^
  |
  = note: closures need an RFC before allowed to be called in constant functions
  = note: calls in constant functions are limited to constant functions, tuple structs and tuple variants
  = note: this error originates in the attribute macro `no_panic` (in Nightly builds, run with -Z macro-backtrace for more info)
